zipline/utils/tradingcalendar.py

```diff
@@ -403,3 +403,9 @@ def get_open_and_closes(trading_days, early_closes, get_open_and_close):
 
 open_and_closes = get_open_and_closes(trading_days, early_closes,
                                       get_open_and_close)
+
+
+def market_minutes_for_day(stamp):
+    index = open_and_closes.index.get_loc(stamp.date())
```

Comment:
Since this body is basically a dupe of TradingEnvironment.get_open_and_close, should we just move that method over to the calendar as well?